umibukela/templates/posters/health_clinic_poster.html

```diff
@@ -5,7 +5,7 @@
   <div class="survey-item col-xs-4">
     {% with q=questions_dict|get:'yes_no_group/all_medication' %}
       <h4 class="question text-center">{{ q.label }}</h4>
-    <div class="response text-center" data-key="yes_no_group/all_medication" data-type="2" data-width="215" data-legend="{% if q.options|length == 3 %}yes/no/na{% else %}yes/no{% endif %}" data-height="112"></div>
```

Comment:
We still indicate yes/no vs yes/other/no - the legend drawing code explicitly draws yes/no. I don't think we need it but cleaning it up isn't driven by an urgent bug right this second and we have other urgent things to fix.